packages/blocks/shuffle/src/components/item.tsx

```diff
@@ -40,44 +42,36 @@ export const Item = forwardRef<HTMLLIElement, ItemProps>(
       listeners,
       dragOverlay,
       readonly,
+      isEntity,
     },
     ref,
   ) => {
     const [isHovered, setIsHovered] = useState(false);
 
     return (
-      <ListItem
+      <SListItem
         ref={ref}
         onMouseOver={() => setIsHovered(true)}
         onMouseOut={() => setIsHovered(false)}
         disablePadding
-        sx={{
-          ...style,
-          marginBottom: 2,
-          opacity: isDragging ? 0 : 1,
-          outlineColor: ({ palette }) => palette.primary.light,
-        }}
+        sx={{ ...style, opacity: isDragging ? 0 : 1 }}
         {...attributes}
       >
-        <Paper
-          sx={{
-            display: "flex",
-            width: 1,
-            paddingX: 2,
-            paddingY: 1,
-            background: ({ palette }) => palette.grey[50],
-            ...paperStyle,
-          }}
-          ref={dragOverlay}
-        >
+        <SPaper sx={paperStyle} ref={dragOverlay}>
+          <Tooltip title={isEntity ? "This item is linked to an entity" : ""}>
+            <SLinkIconWrapper>
+              {isEntity && <LinkIcon fontSize="small" />}
+            </SLinkIconWrapper>
+          </Tooltip>
```

Comment:
```suggestion
        {isEntity ? (
            <Tooltip title="This item is linked to an entity">
              <SLinkIconWrapper>
                <LinkIcon fontSize="small" />
              </SLinkIconWrapper>
            </Tooltip>
          ) : null}
```

Reply:
The wrapper should be rendered even if it's empty, since it has a fixed width, and it aligns the start of the each item's text content.
<img width="227" alt="image" src="https://user-images.githubusercontent.com/39553853/189568437-7ab27582-8ddb-4274-aade-93e0eefc819b.png">

Reply:
I had noticed that, but I think it's odd we're rendering an empty Tooltip for spacing. We could give a conditional spacing to the Textfield next to it, what do you think?

Reply:
Makes sense, thanks for the suggestion!

Reply:
Updated here https://github.com/hashintel/hash/pull/1033/commits/b37cab1789f1d7571cb91235dce1a187fbeffe25